src/chainstate/coordinator/mod.rs

```diff
@@ -380,14 +381,15 @@ pub fn get_next_recipients<U: RewardSetProvider>(
 ///           * Was PoX anchor block known?
 pub fn get_reward_cycle_info<U: RewardSetProvider>(
     burn_height: u64,
+    num_sortitions: u64,
```

Comment:
Can't this value be queried from the `sort_db`, instead of being passed via an argument?

Reply:
It could, the intention was to avoid a query, since we have the data available (and beside tests, this method is invoked once or twice)

Reply:
I'd much rather prefer the query, since then we never have to worry about the `num_sortitions` argument being wrong (neither now nor in the future).

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2233/commits/321fb172ae50c6a4921169d43725521e536fc19d